### PR TITLE
Jd/fixes area balancing models

### LINKED ISSUE #1107: Implement slack variables in the UpperBoundFeedforward constraints
Currently the variables are created but they are not added to the constraint. This is correctly done in the LowerBoundFeedforward

### PR DESCRIPTION
**Pull request by jd-lara**
closes #1107

### CODE REVIEW

File: src/feedforward/feedforward_arguments.jl [hunk 1]
```diff
@@ -135,11 +136,12 @@ function _add_feedforward_arguments!(
     parameter_type = get_default_parameter_type(ff, T)
     add_parameters!(container, parameter_type, ff, model, devices)
     if get_slacks(ff)
-        add_feedforward_slack_variables!(
+        _add_feedforward_slack_variables!(
```
Comment (rodrigomha) [inline src/feedforward/feedforward_arguments.jl:139]: We should test that the Slack FFs are working

File: src/feedforward/feedforward_arguments.jl [hunk 2]
```diff
@@ -175,7 +178,7 @@ function _add_feedforward_arguments!(
     if get_slacks(ff)
         _add_feedforward_slack_variables!(
             container,
-            LowerBoundFeedForwardSlack,
+            LowerBoundFeedForwardSlack(),
```
Comment (rodrigomha) [inline src/feedforward/feedforward_arguments.jl:181]: It seems that the LL FF Slacks are being tested but not the UB

File: src/feedforward/feedforward_arguments.jl (hunk 1 above)
Comment (jd-lara) [reply src/feedforward/feedforward_arguments.jl:139]: true

File: src/parameters/update_parameters.jl [hunk 3]
```diff
@@ -403,7 +408,7 @@ function update_container_parameter_values!(
     # if the keys have strings in the meta fields
     parameter_array = get_parameter_array(optimization_container, key)
     parameter_attributes = get_parameter_attributes(optimization_container, key)
-    _update_parameter_values!(parameter_array, parameter_attributes, T, model, input)
+    _update_parameter_values!(parameter_array, parameter_attributes, U, model, input)
```
Comment (rodrigomha) [inline src/parameters/update_parameters.jl:411]: Why is this Update Parameter for FixValueParameter is not being tested?

**Review by rodrigomha — COMMENTED** [review]
This looks good. We may need to QA/QC the DCP/PTDF models for AreaInterchange.

Besides the testing, this should be fine for generalizing

File: src/parameters/update_parameters.jl (hunk 3 above)
Comment (jd-lara) [reply src/parameters/update_parameters.jl:411]: Tests added in [192dc3c](https://github.com/NREL-Sienna/PowerSimulations.jl/pull/1130/commits/192dc3c02a1fbc41631abf9d6f41fca946efd826)

File: src/feedforward/feedforward_arguments.jl (hunk 1 above)
Comment (jd-lara) [reply src/feedforward/feedforward_arguments.jl:139]: tests added in [192dc3c](https://github.com/NREL-Sienna/PowerSimulations.jl/pull/1130/commits/192dc3c02a1fbc41631abf9d6f41fca946efd826)

**Comment by rodrigomha** [comment]
It also:
Closes https://github.com/NREL-Sienna/PowerSimulations.jl/issues/1100
Closes https://github.com/NREL-Sienna/PowerSimulations.jl/issues/1103
Closes https://github.com/NREL-Sienna/PowerSimulations.jl/issues/1134
Closes https://github.com/NREL-Sienna/PowerSimulations.jl/issues/1135
via #1136

**Review by rodrigomha — APPROVED** [review]

**Pull request merged**